docs/components/concepts/process-instance-migration.md

```diff
@@ -141,6 +141,129 @@ The called process instance is not changed when migrating the call activity.
 
 You can migrate a called process instance in the same way as a regular process instance.
 
+## Dealing with catch events
+
+An exception to changing the process instance state is specific to catch events.
+This is necessary to ensure that the process instance can be executed according to the new process definition.
+This allows you to add or remove catch events from an active element.
+At the same time, you can leave an existing catch event unchanged.
+This section explains how to deal with catch events when migrating a process instance.
+
+You decide what happens to the associated event subscription through the mapping instructions for the catch events:
+
+- Migrate catch events: if a catch event is mapped, the associated subscription is migrated.
+- Remove catch events: if a catch event in the source process is not mapped, then the associated subscription is closed during migration.
+- Add catch events: if a catch event of the target process is not the target of a mapping instruction, then a new subscription is opened during migration.
+
+### Migrate catch events
+
+Catch events can be migrated by providing a mapping instruction between the source and the target catch event.
+Providing a mapping between catch events ensures that the behaviour of the event subscription in the source process is preserved after the migration.
+This is especially useful when a subscription behaviour is changed in the new version of the process definition, but you want to keep the catch event behaviour of the running processes the same after the migration.
```

Comment:
🔧 I suggest removing this sentence. The sentence below informs us to read the next sections to understand when this is useful. The previous sentences have already provided the information that the event subscription is preserved.